alexandria/core/serializers.py

```diff
@@ -197,6 +197,17 @@ def validate(self, *args, **kwargs):
 
         return validated_data
 
+    def create(self, validated_data):
+        created = super().create(validated_data)
+
+        # Update document's modified when a file is created.
+        created.document.modified_at = created.modified_at
```

Comment:
This field has `auto_now=True`, so it's not necessary to set it manually.

Reply:
Removed the line